frontend/lib/services/truck_service.dart

```diff
@@ -2,7 +2,7 @@ import "package:http/http.dart" as http;
 import 'dart:convert';
 import "../models/truck_model.dart";
 
-var host = 'http://0.0.0.0:5000';
+var host = 'http://10.0.2.2:5000';
```

Comment:
maybe change the host back to 0.0.0.0 as 10.0.2.2 is your IP address? It would be a good idea to add a comment here or README.md to remind others to change host to their IP address if necessary.